README.md

```diff
@@ -1,8 +1,64 @@
-# Test Repo
+# EnChord
 
-The chopsticks are the best female developer duo.
+**EnChord** is a perfect tool to develop your music dictation skills. Challenge yourself with different levels of quizzes and follow your progress by checking your highest score and average test results. You can also review what are the frequently mistaken interval types.
 
-## Scripts
+## Table of contents
+
+1. [How it works](https://github.com/TheChopsticks/EnChord/edit/main/README.md#how-it-works)
+2. [Future feature](https://github.com/TheChopsticks/EnChord/edit/main/README.md#future-features)
+3. [How EnChord was built](https://github.com/TheChopsticks/EnChord/edit/main/README.md#how-enchord-was-built)
+4. [Installation](https://github.com/TheChopsticks/EnChord/edit/main/README.md#installation)
+5. [Credits](https://github.com/TheChopsticks/EnChord/edit/main/README.md#credits)
+6. [License](https://github.com/TheChopsticks/EnChord/edit/main/README.md#license)
+7. [Badges](https://github.com/TheChopsticks/EnChord/edit/main/README.md#badges)
+
+## How it works
+
+#### Try 3 different levels
+
+- **EnChord** has 3 different levels, Easy, Intermediate, and Hard.
+
+#### What are the differences
+
+- **For enthusiastic beginners:**
+
+  - 2 random notes will be played only upwards to help you guess the answer.
+  - There is an unlimited number of getting hints for you to listen to full scale between 2 notes
+
+- **For intermediate-level players with basic skills:**
+
+  - 2 random notes will be played either upwards or downwards
+  - There is a limited number of hints
+
+- **For skilled musicians:**
+  - 2 random notes will be played TOGETHER
+  - There is no hint available.
+
+## Future features
+
+#### Review and follow your progress:
+
+- After each quiz set, you can review which questions you got wrong and listen to them again!
+- Check your highest score and the average score of previously played games.
+- Check your weakness by reviewing the most wrong interval types.
+
+#### Language support:
+
+- Mandarin(普通话), Korean(한국어), Hungarian(Magyar)
```

Comment:
```suggestion
- Hungarian (Magyar), Korean (한국어), Traditional Chinese (繁體中文)
```
Nitpick on the terminology here ("Mandarin" is usually used to refer to the spoken language) and a small suggestion to use alphabetic order so we don't show unintended preference 🙃

Reply:
Actually, I wanted to ask you what to write for the Chinese part. I googled them but still wasn't 100% sure. Thank you. Also now I know, it is a norm to use alphabetic order 👍